src/tools/rest-backend/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+find_package (CppCMS)
+find_package (Boost 1.50 COMPONENTS system filesystem thread REQUIRED)
```

Comment:
After thinking about it again, I can give you an answer. I'm using boost for synchronization (database cache) as well as the creation of temporary files. I tried using only the CppCMS package, but it didn't want to configure anymore (cmake). So I'll keep the import.